src/shared/style/base.jsx

```diff
@@ -0,0 +1,9 @@
+import { injectGlobal } from 'emotion'
+import ZillaSlab from './ZillaSlab-Light.woff'
```

Comment:
Not sure if this makes sense from a style POV, but I usually namespace as if these are constants when I'm bringing in fonts, so maybe `ZILLA_SLAB`. This import is actually just a generated path string, iirc. Open to what you have as well, just wanted to bring it up.

Reply:
Sure, I can do that. Was just copying what was specified initially in the base styles js file, iirc

Reply:
yep, I definitely didn't do this right when I added zilla slab :)